src/main/java/com/github/vkorobkov/jfixtures/sql/SqlBase.java

```diff
@@ -33,6 +36,12 @@ default void insertRow(Appender appender, InsertRow insertRow) {
         appender.append("INSERT INTO ", table, " (", columns, ") VALUES (", values, ");\n");
     }
 
+    @SneakyThrows
+    @Override
+    default void addCustomSql(Appender appender, CustomSql customSql) {
+        appender.append(customSql.getInstruction().replace(TABLE_NAME_PLACEHOLDER, customSql.getTable()), "\n");
```

Comment:
Let's move the replacement logic into `CustomSql`? Probably we can just to the replacement right in the constructor of it because it consumes both table name and sql.